### ISSUE TITLE
Plague Tale requiem : Popup shows but FG option is invisible. Hardware GPU acceleration is enabled

### ISSUE THREAD

**Issue by Edwin-Joe** [Nukem9/dlssg-to-fsr3#118]

**Comment by AJTHEVAMP** [comment]
Have u applied files correctly as mine is working fine with rtx 3060

**Comment by Onkelpuh** [comment]
have the same problem
Win10 RTX3060

**Comment by Ripthorn** [comment]
works for me on gamepass version. Try with the non dlsstweaks version.

**Comment by vlbastos** [comment]
Just tested version 0.60 in a mobile RTX 3070, it works good. Side effects: the artifacts around characters get worse when screen moves (motion blur) (they exist even with frame generator off), subtitles and HUD tremble with movement. Looks really great anyways. Jumped from 37 fps to 67 fps in tutorial.
![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/50724237/f31926e0-0239-4f0d-b516-bb045311af49)

**Comment by Edwin-Joe** [comment]
> Have u applied files correctly as mine is working fine with rtx 3060

yes, run disablenv....checks, copy dlls to the exe folder and run game.

**Comment by Edwin-Joe** [comment]
> Just tested version 0.60 in a mobile RTX 3070, it works good. Side effects: the artifacts around characters get worse when screen moves (motion blur) (they exist even with frame generator off), subtitles and HUD tremble with movement. Looks really great anyways. Jumped from 37 fps to 67 fps in tutorial. Xbox Gamepass PC version (C:\XboxGames\A Plague Tale- Requiem - Windows\Content\APT2_WinStore.x64.Submission.exe) ![image](https://private-user-images.githubusercontent.com/50724237/291753316-f31926e0-0239-4f0d-b516-bb045311af49.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMwNTg0MjQsIm5iZiI6MTcwMzA1ODEyNCwicGF0aCI6Ii81MDcyNDIzNy8yOTE3NTMzMTYtZjMxOTI2ZTAtMDIzOS00ZjBkLWI1MTYtYmIwNDUzMTFhZjQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIwVDA3NDIwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyMDRhMzMzOWRmNDZmMjAyYWRlYWMzYzNiY2UyNmMyZmJmMmUxMzc5ZWExNWNkYTNiNDUwNDQ1NmNlMzJhNzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bgJevRu8AgplXMyyAnizm5FsBUX2CIGdB2bxx2yIp_E)

coul you plrase point out the steps? I tried as per instructions and doesnt work

**Comment by vlbastos** [comment]
> > Just tested version 0.60 in a mobile RTX 3070, it works good. Side effects: the artifacts around characters get worse when screen moves (motion blur) (they exist even with frame generator off), subtitles and HUD tremble with movement. Looks really great anyways. Jumped from 37 fps to 67 fps in tutorial. Xbox Gamepass PC version (C:\XboxGames\A Plague Tale- Requiem - Windows\Content\APT2_WinStore.x64.Submission.exe) ![image](https://private-user-images.githubusercontent.com/50724237/291753316-f31926e0-0239-4f0d-b516-bb045311af49.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMwNTg0MjQsIm5iZiI6MTcwMzA1ODEyNCwicGF0aCI6Ii81MDcyNDIzNy8yOTE3NTMzMTYtZjMxOTI2ZTAtMDIzOS00ZjBkLWI1MTYtYmIwNDUzMTFhZjQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjIwVDA3NDIwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyMDRhMzMzOWRmNDZmMjAyYWRlYWMzYzNiY2UyNmMyZmJmMmUxMzc5ZWExNWNkYTNiNDUwNDQ1NmNlMzJhNzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bgJevRu8AgplXMyyAnizm5FsBUX2CIGdB2bxx2yIp_E)
> 
> coul you plrase point out the steps? I tried as per instructions and doesnt work

1. Run "DisableNvidiaSignatureChecks.reg" in any folder and accept the merge
2. Copy "dlssg_to_fsr3_amd_is_better.dll" and "nvngx.dll" to the same folder where "APT2_WinStore.x64.Submission.exe" is
3. Run the game.

REMEMBER TO ENABLE HARDWARE GPU SCHEDULING IN WINDOWS SETTINGS AND SET V-SYNC OFF INGAME.

**Comment by izyspania** [comment]
I think it doesnt work in the cracked version of the game.

The .exe is APlagueTaleRequiem_x64.exe
---
dlssg-to-fsr3 is loaded. AMD FSR 3 Frame Generation will replace Nvidia DLSS-G Frame Generation.

Please note this is experimental software and is subject to bugs and/or crashes. This does NOT represent a native implementation of AMD's FSR 3.

Report issues at https://github.com/Nukem9/dlssg-to-fsr3

WARNING: DO NOT USE IN MULTIPLAYER GAMES.
NVSDK_NGX_D3D12_Init_Ext
---

![image](https://github.com/Nukem9/dlssg-to-fsr3/assets/93366907/1e44f082-bab0-4089-97c3-d560c243ebe6)

**Comment by vlbastos** [comment]
Then please state in the topic title that the problem is with the cracked, pirated game. I don't think the dev will support those.

**Comment by vlbastos** [comment]
> I think it doesnt work in the cracked version of the game. Not even DLSSTweaks seem to work in this version.
> 
> ## The .exe is APlagueTaleRequiem_x64.exe

Then please state in the topic title that the problem is with the cracked, pirated game. I don't think the dev will support those.

**Comment by izyspania** [comment]
> > I think it doesnt work in the cracked version of the game. Not even DLSSTweaks seem to work in this version.
> > ## The .exe is APlagueTaleRequiem_x64.exe
> 
> Then please state in the topic title that the problem is with the cracked, pirated game. I don't think the dev will support those.

Its not my topic , i was just testing some games myself for the sake of testing and i took this cracked one only for the sake of testing (because i dont have it and i dont intend to play it anyway) and i noticed that it doesnt work.

**Comment by Folka-1** [comment]
I faced the same problem and I didn't find any solution but there is one way the worked with me :
1 - Delete any files you added to the game
2 - take a copy of your saves and delete all files in the folder of saves 
       Save file location " C:\Users\Public\Documents\Steam\CODEX\1182900\remote "
3 - make sure go to setting files and open it with notepad and set int Video/ " DLSSG 0 " to " DLSSG 1 "
4 - Download the file " fsr2.2 to FSR3 " not " dlssg-to-fsr3 " 
5 - copy all files to the game folder where " .exe " file located
run the game and enter a new game then check setting you will still can't change the value of FG on or off but it is on it raised the FPS from 37 to 60+ 
6 - close the game and recopy the save file only to its location " slot0n " only 
7 - Enjoy
NOTE : don't turn off the dlss because there is many glitches and FG doesn't work well so just put it on " ultra quality "
